src/test/java/de/tum/cit/aet/artemis/programming/icl/LocalVCLocalCIIntegrationTest.java

```diff
@@ -418,6 +421,15 @@ void testFetchPush_studentAssignmentRepository_afterDueDate() throws Exception {
         // Instructor
         localVCLocalCITestService.testFetchSuccessful(assignmentRepository.localGit, instructor1Login, projectKey1, assignmentRepositorySlug);
         localVCLocalCITestService.testPushSuccessful(assignmentRepository.localGit, instructor1Login, projectKey1, assignmentRepositorySlug);
+
+        vcsAccessLogRepository.findAllByParticipationId(participation.getId());
+        var vcsAccessLogs = vcsAccessLogRepository.findAllByParticipationId(participation.getId());
```

Comment:
The duplicated call seems to have no effect.

```suggestion
        var vcsAccessLogs = vcsAccessLogRepository.findAllByParticipationId(participation.getId());
```

Reply:
good point, missed that